src/sdkV2.ts

```diff
@@ -53,36 +49,53 @@ export default class CegaEvmSDKV2 {
     return chainConfig;
   }
 
-  loadCegaEntry(): ethers.Contract {
+  loadAddressManager(): ethers.Contract {
     return new ethers.Contract(
-      this._cegaEntryAddress,
+      this._addressManagerAddress,
```

Comment:
to `loadAddressManager` you need to use that ABI

Reply:
updated ABI :)